jerry-core/ecma/builtin-objects/ecma-builtin-math.c

```diff
@@ -282,6 +344,22 @@ ecma_builtin_math_dispatch_routine (uint16_t builtin_routine_id, /**< built-in w
         x = DOUBLE_TO_ECMA_NUMBER_T (log (x));
         break;
       }
+
+#if ENABLED (JERRY_ES2015_BUILTIN)
```

Comment:
Please remove the empty lines around the preprocessor directives, here and below.